detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/BracesOnIfStatements.kt

```diff
@@ -0,0 +1,178 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import io.gitlab.arturbosch.detekt.api.config
+import io.gitlab.arturbosch.detekt.api.internal.Configuration
+import org.jetbrains.kotlin.psi.KtBlockExpression
+import org.jetbrains.kotlin.psi.KtExpression
+import org.jetbrains.kotlin.psi.KtIfExpression
+
+/**
+ * This rule detects `if` statements which do not comply with the specified rules.
+ * Keeping braces consistent would improve readability and avoid possible errors.
+ *
+ * SingleLine if-statement has no '\n':
+ * if (a) b else c
+ * MultiLine if-statement has at least one '\n':
+ * if (a) b
+ * else c
```

Comment:
```suggestion
 * SingleLine if-statement has no '\n':
 * ```
 * if (a) b else c
 * ```
 * MultiLine if-statement has at least one '\n':
 * ```
 * if (a) b
 * else c
 * ```
```

Reply:
right